web/static/js/sidebar.js

```diff
@@ -67,3 +78,34 @@ function toggleArrows(type) {
   toggleClasses(select(".select-" + type + "-filters > h5 > .fa-up"), ["filter-arrow-hide"]);
   toggleClasses(select(".select-" + type + "-filters > h5 > .fa-down"), ["filter-arrow-hide"]);
  }
+
+function updateSelected(form) {
+  var formData = new FormData(form);
+
+  for (var value of formData.entries()) {
```

Comment:
Not too sure about the browser compatibility of `for... of` loops. Couldn't find info on MDN or CanIUse, but [here](https://docs.microsoft.com/en-us/scripting/javascript/reference/for-dot-dot-dot-of-statement-javascript) it says `Not supported in the following document modes... Internet Explorer 8 standards, Internet Explorer 9 standards, Internet Explorer 10 standards`

I'm a little unclear how supported this is tbh. You could probably write something similar with a `for` loop or `for...in` loop or something.